### PR TITLE
WFP-750 get probation status

### PR DESCRIPTION
**Pull request by markreesmoj**

### CODE REVIEW

File: src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsallocations/integration/IntegrationTestBase.kt [hunk 1]
```diff
@@ -135,24 +134,43 @@ abstract class IntegrationTestBase {
...
+      response().withContentType(APPLICATION_JSON).withBody(singleActiveAndInactiveConvictionsResponse())
+    )
+  }
+
   protected fun allDeliusResponses(crn: String) {
     singleActiveConvictionResponse(crn)
     singleActiveInductionResponse(crn)
     offenderSummaryResponse(crn)
     tierCalculationResponse(crn)
+    singleActiveConvictionResponse(crn)
```
Comment (carlov20) [inline src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsallocations/integration/IntegrationTestBase.kt:173]: this mock has already been setup on line 169, is it required?

File: src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsallocations/integration/unallocatedcases/EventListenerTest.kt [hunk 2]
```diff
@@ -24,11 +23,13 @@ class EventListenerTest : IntegrationTestBase() {
     noActiveInductionResponse(crn)
     offenderSummaryResponse(crn)
     tierCalculationResponse(crn)
+    singleActiveConvictionResponse(crn)
```
Comment (carlov20) [inline src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsallocations/integration/unallocatedcases/EventListenerTest.kt:26]: similar with this mock, already on line 22

**Comment by carlov20** [comment]
README needs updating as well if we still want the format of the CSV in there

File: src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsallocations/integration/IntegrationTestBase.kt (hunk 1 above)
Comment (markreesmoj) [reply src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsallocations/integration/IntegrationTestBase.kt:173]: there are now up to 3 requests for convictions

Comment (markreesmoj) [reply src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsallocations/integration/IntegrationTestBase.kt:173]: I could make the setup clearer though, as the first is for active convictions only

**Review by carlov20 — APPROVED** [review]

**Pull request merged**